contributors/design-proposals/storage/raw-block-pv.md

```diff
@@ -528,6 +528,41 @@ spec:
         name: my-nginx-data 
 	readOnly: false
 ```
+## UC9:
+
+DESCRIPTION:
+* A user wishes to read data from a read-only raw block device, such as a database for analytics processing. 
+
+ADMIN:
+* Admin creates a 1 block devices and with the intention consumption of read-only. Admin should label the PV
```

Comment:
Not sure if we need to suggest access mode RO.  You could have a producer/consumer use case where one pod writes, and another reads.

Reply:
@msau42 hum...are we suggesting we set different container runtime values for the same pvc/pv?

Reply:
I believe the container runtime access modes only dictate the permission of the device in the container, not on the host.  But that needs to be verified.

Reply:
@msau42 that is accurate --- but shouldn't we request admin to use best practices to indicate if the volume is statically provisioned (as detailed in thise use case) to say it's RO ?

Reply:
I'll just remove.. @msau42 @mtanino